studips/src/component/PostModal.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '../App.css';
+
+function PostModal({ isPostModalVisible }) {
+	return isPostModalVisible ? (
+		<div className='overlay'>
```

Comment:
change tabs for spaces, and recheck indentation

Reply:
Fixed as well.